raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -1149,12 +1187,58 @@ def handle_lock_expired(
     return TransitionResult(mediator_state, events)
 
 
+def handle_node_change_network_state(
+        mediator_state: MediatorTransferState,
+        state_change: ActionChangeNodeNetworkState,
+        channelidentifiers_to_channels: typing.ChannelMap,
+        pseudo_random_generator: random.Random,
+        block_number: typing.BlockNumber,
+) -> TransitionResult:
+    """ If a certain node comes online:
+    1. Check if a channel exists with that node
+    2. Check that this channel is a route, check if the route is valid.
+    3. Check that the transfer was stuck because there was no route available.
+    4. Send the transfer again to this now-available route.
+    """
+    if state_change.network_state != NODE_NETWORK_REACHABLE:
+        return TransitionResult(mediator_state, list())
+
+    # Get route whose node_address matches the newly reachable node
```

Comment:
Please, don't add comments that say the same thing as the code.

IMO, the docstring is sufficient and better written in this case.